dev/GeneratedExperimental/TitleBar.properties.cpp

```diff
@@ -92,7 +92,7 @@ void TitleBarProperties::EnsureProperties()
                 winrt::name_of<winrt::TitleBar>(),
                 false /* isAttached */,
                 ValueHelper<winrt::hstring>::BoxedDefaultValue(),
-                nullptr);
```

Comment:
Where are theses generated files changes coming from? I don't see changes to the input file (idl) or the code gen logic.

Reply:
Looks like the generated files were not updated with this PR: #6326. 

So these changes are unrelated to the PackageES Security and Compliance work. But it should not be a big deal to include them here.